File: rust/src/k08_1sub.rs
pub fn get_max_value(v1: i32, v2: i32, v3: i32) -> i32 {
    let mut temp = if v1 >= v2 {
        v1
    } else {
        v2
    };

    temp = if temp >= v3 {
        temp
    } else {
        v3
    };

    temp
}

pub fn get_min_value(v1: i32, v2: i32, v3: i32) -> i32 {
    let mut temp = if v1 <= v2 {
        v1
    } else {
        v2
    };

    temp = if temp <= v3 {
        temp
    } else {
        v3
    };

    temp
}

File: SwiftProject/SwiftProject/K08_1sub.swift
//
//  K08_1sub.swift
//  SwiftProject
//
//  Created by 周哲輝 on 2019/05/11.
//  Copyright © 2019 周哲輝. All rights reserved.
//

import Foundation

public func GetMaxValue(v1: Int, v2: Int, v3: Int) -> Int {
    var temp = v1 >= v2 ? v1 : v2
    temp = temp >= v3 ? temp : v3
    return temp
}

public func GetMinValue(v1: Int, v2: Int, v3: Int) -> Int {
    var temp = v1 < v2 ? v1 : v2
    temp = temp < v3 ? temp : v3
    return temp
}
